source/command-logging-and-monitoring/tests/logging/unacknowledged-write.yml

```diff
@@ -5,6 +5,7 @@ schemaVersion: "1.16"
 createEntities:
   - client:
       id: &client client
+      useMultipleMongoses: false
```

Comment:
Note: PHP doesn't yet run logging tests, so I'd defer to another driver to test this.